cpp/benchmarks/CMakeLists.txt

```diff
@@ -13,10 +15,9 @@ function(ConfigureBench CMAKE_BENCH_NAME CMAKE_BENCH_SRC)
     target_link_libraries(${CMAKE_BENCH_NAME} benchmark benchmark_main pthread cudf )
     set_target_properties(${CMAKE_BENCH_NAME} PROPERTIES
                             RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/gbenchmarks")
+    set(BENCHMARK_LIST ${BENCHMARK_LIST} ${CMAKE_BENCH_NAME} CACHE INTERNAL "BENCHMARK_LIST")
 endfunction(ConfigureBench)
 
-option(CMAKE_ENABLE_BENCHMARKS "Enable building cuDF benchmarks" OFF)
```

Comment:
So is there no way to disable building them (e.g. when building locally) now?

Reply:
This variable was not used in this CMake file at all. I removed what seemed like dead code (see the change in the CMake file in /tests too). 
You can specify whether to build benchmarks in CMake using -DBUILD_BENCHMARKS flag now.